packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -946,13 +946,6 @@ abstract class RouteTransitionRecord {
   /// route to indicate that the route should be completed with the provided
   /// result and removed from the [Navigator] without an animated transition.
   void markForComplete([dynamic result]);
-
-  /// Marks the [route] to be removed without transition.
-  ///
-  /// During [TransitionDelegate.resolve], this can be called on an exiting
-  /// route to indicate that the route should be removed from the [Navigator]
-  /// without completing and without an animated transition.
-  void markForRemove();
```

Comment:
We should mark this deperecated and the implementation calls the markForComplete directly.

Reply:
I think this will fail ci since there is no default implementation and subclass doesn't override this

maybe just 
```
void markForRemove() => markForComplete();
```